### ISSUE TITLE
FeatureImp does not work for classisifaction models

### ISSUE THREAD

**Issue by aiwalter** [christophM/iml#66]
**Exemplary classification case with binary classification:**
`library("mlbench")
data("PimaIndiansDiabetes")
data = PimaIndiansDiabetes
rf = randomForest(diabetes~.,data = data,trees = 100)

train_data = data[ , -which(names(data) %in% "diabetes")]

model = Predictor$new(model = rf, data = train_data, y = data$diabetes)

explanation = FeatureImp$new(model, loss = "mse", n.repetitions = 1, run = TRUE)

**Error:**
`Error in if (self$original.error == 0) { : 
  missing value where TRUE/FALSE needed
In addition: Warning message:
In Ops.factor(actual, predicted) : ‘-’ not meaningful for factors`

**Comment by christophM** [comment]
Please check the example in ?FeatureImp
Basically, you have to set `type = "prob"` in Predictor$new()
